modules/running-your-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+[id="running-your-dynamic-plugin"]
+= Running your dynamic plug-in
+You can run the plugin using a local development environment. The OpenShift console will run in a container connected to the cluster you have logged into.
+
+[id="run-plug-in-locally"]
+== Run your plug-in locally
+
+.Prerequisites
+* You must have the OpenShift CLI link:https://mirror.openshift.com/pub/[(`oc`)] installed.
+* You must have an OpenShift cluster running.
+* You must have link:https://www.docker.com/[Docker] or link:https://podman.io/[Podman] installed.
```

Comment:
We should also check here about how we want to handle Docker vs podman.

Reply:
I added mac/RHEL but I am not sure if that is what you are looking for.

Reply:
[Here]( https://docs.openshift.com/container-platform/4.9/openshift_images/index.html#images-about_overview-of-images) is how we have docker/podman in other places of the docs. It looks like users can choose either or. Is that the case for this?

Reply:
Yes, it's possible for developers to use either, and the commands are otherwise the same.

Reply:
Shall we say Docker/Podman should be running? @opayne1 
```
* You must have link:https://www.docker.com/[Docker] or link:https://podman.io/[Podman] installed and running
```